tox.ini

```diff
@@ -8,6 +8,7 @@ minversion = 3.3.0
 deps =
     pytest~=6.0
     pytest-cov~=2.0
+    pytest-mock~=3.0
```

Comment:
note to myself: apparently we do not have `tests` extra_requires and all test requirements are set in `tox` making it the only kosher way to run tests.